src/modules/createChannel/CreateChannel.tsx

```diff
@@ -1,128 +1,340 @@
 // This is the Parent component for the Create Channel flow.
+// React + web3 essentials
+import { useEffect, useState } from 'react';
+
+// Third party libraries
+import { useFormik } from 'formik';
+import { ethers } from 'ethers';
+
```

Comment:
<img width="1509" alt="Screenshot 2024-07-16 at 3 35 44 PM" src="https://github.com/user-attachments/assets/a7f9bb06-a48d-4915-af84-b07a058478e9">

This needs alignment fixes

Reply:
This will get fixed when we will merge new channel dashboard into this flow also.